### ISSUE TITLE
Magento 2.0.9 stops sending E-Mails

### ISSUE THREAD

**Issue by flipflap** [magento/magento2#6163]
### Preconditions

1. Magento 2.0.9, bofore it worked well on Magento 2.0.8
2. Cronjobs ok, it worked before
3. I tested with different order E-Mails
### Steps to reproduce

1. Make an Order
2. Checkout and payment
### Expected result

1. Sale confirmation E-Mail should be sent
### Actual result

1. Nothing, not E-Mail

**Comment by wert2all** [comment]
I cannot reproduce the issue.
Please provide more preconditions to reproduce this issue. Is there something in the Magento logs?

**Comment by flipflap** [comment]
My hoster coudn't find anything in the Server Log. It is like nothing happen.

I checked Magento Logs and magento.cron.log says: Ran jobs by schedule.

I found this in system.log, but I am not sure if this has something to do with it:

`[2016-08-15 10:11:27] main.CRITICAL: Broken reference: the 'global.search' tries to reorder itself towards 'notification.messages', but their parents are different: 'header.inner.right' and 'header' respectively. [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'global_notices' element cannot be added as child to 'global.notices', because the latter doesn't exist [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'logo' element cannot be added as child to 'header', because the latter doesn't exist [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'global.search' element cannot be added as child to 'header', because the latter doesn't exist [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'user' element cannot be added as child to 'header', because the latter doesn't exist [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'menu' element cannot be added as child to 'page.menu', because the latter doesn't exist [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'breadcrumbs' element cannot be added as child to 'page.breadcrumbs', because the latter doesn't exist [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'formkey' element cannot be added as child to 'page.formkey', because the latter doesn't exist [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'page.title' element cannot be added as child to 'main.top', because the latter doesn't exist [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'messages' element cannot be added as child to 'page.messages', because the latter doesn't exist [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'page.actions.toolbar' element cannot be added as child to 'page.main.actions', because the latter doesn't exist [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'legal' element cannot be added as child to 'footer', because the latter doesn't exist [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'system_messages' element cannot be added as child to 'notifications', because the latter doesn't exist [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'unread_system_messages' element cannot be added as child to 'notifications', because the latter doesn't exist [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'notification_window' element cannot be added as child to 'notifications', because the latter doesn't exist [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'notification.messages' element cannot be added as child to 'header', because the latter doesn't exist [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'header.inner.left' element cannot be added as child to 'header', because the latter doesn't exist [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'header.inner.right' element cannot be added as child to 'header', because the latter doesn't exist [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'logo' tries to reorder itself towards '', but their parents are different: 'header' and '' respectively. [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'global.search' tries to reorder itself towards 'logo', but their parents are different: 'header' and '' respectively. [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'user' tries to reorder itself towards '', but their parents are different: 'header' and '' respectively. [] []
[2016-08-15 10:13:25] main.CRITICAL: Broken reference: the 'page.actions.toolbar' tries to reorder itself towards '', but their parents are different: 'page.main.actions' and ''`
